Content.Client/Doors/DoorSystem.Airlock.cs

```diff
@@ -33,14 +30,14 @@ private void OnComponentStartup(EntityUid uid, AirlockComponent comp, ComponentS
         ((Animation)door.OpeningAnimation).AnimationTracks.Add(new AnimationTrackSpriteFlick()
             {
                 LayerKey = DoorVisualLayers.BaseUnlit,
-                KeyFrames = { new AnimationTrackSpriteFlick.KeyFrame(comp.OpeningSpriteState, 0f) },
+                KeyFrames = {new AnimationTrackSpriteFlick.KeyFrame(comp.OpeningSpriteState, 0f)},
```

Comment:
Rider's auto-format reverts this whitespace change.

Reply:
Understood. Can you point me to where this was described so I can get the rationale?